From 55802fd187b844f0c114c1657b18482bfc80ec51 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 2 Jul 2019 02:52:31 +0000
Subject: [PATCH] fix out of tree build failure

Since commit [116da95 Add the '--version' option to bs_calc.py] applied,
while build out of tree, there is a install failure
...
|install -m0755 ../../git/tools/bs_calc.py /usr/bin/bscalc
|install: cannot stat '../../git/tools/bs_calc.py': No such file or directory
...

The generated bs_calc.py locates in builddir rather than srcdir

Upstream-Status: Submitted [https://github.com/storaged-project/libbytesize/pull/59]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 tools/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/Makefile.am b/tools/Makefile.am
index 08223a8..ae22fed 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -5,7 +5,7 @@ dist_man1_MANS = bscalc.man
 
 install-exec-local:
 	install -d ${DESTDIR}${bindir}
-	install -m0755 ${srcdir}/bs_calc.py ${DESTDIR}${bindir}/bscalc
+	install -m0755 ${builddir}/bs_calc.py ${DESTDIR}${bindir}/bscalc
 
 uninstall-local:
 	rm ${DESTDIR}${bindir}/bscalc
-- 
2.17.1

